### PR TITLE
Update translations in `Files.ja-JP.xlf`.

### PR DESCRIPTION
**Pull request by h1g0**
**Resolved / Related Issues**
Items resolved / related issues by this PR.

**Details of Changes**
Add details of changes here.
- Added Japanese translations for recently added texts. ( FYI: Recent contributors to Japanese translations @inductor @akihironagai  )

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: src/Files/MultilingualResources/Files.ja-JP.xlf [hunk 1]
```diff
@@ -3451,71 +3451,71 @@ App Center を使用して、アプリの使用状況を追跡し、バグを見
...
         </trans-unit>
         <trans-unit id="SetAsBackground" translate="yes" xml:space="preserve">
           <source>Set as desktop background</source>
-          <target state="new">Set as desktop background</target>
+          <target state="translated">デスクトップ背景に設定</target>
         </trans-unit>
         <trans-unit id="SetFilesAsDefaultWarning" translate="yes" xml:space="preserve">
           <source>Do not remove Files from your device without turning off this option. Not doing so will prevent you from opening File Explorer.</source>
-          <target state="new">Do not remove Files from your device without turning off this option. Not doing so will prevent you from opening File Explorer.</target>
+          <target state="translated">このオプションをオフにせずデバイスからFiles を削除しないでください。このオプションをオフにしてからFiles を削除しないと、ファイル エクスプローラーを開くことができなくなります。</target>
```
Comment (inductor) [inline src/Files/MultilingualResources/Files.ja-JP.xlf:3486]: There's blank only after "Files", is this intentional?

Comment (h1g0) [reply src/Files/MultilingualResources/Files.ja-JP.xlf]: Oh, that was just a simple mistake, and I fixed it.
I referred to other places and inserted a space before "Files" as well.

**Review by inductor — APPROVED** [review]
LGTM

**Review by d2dyno1 — APPROVED** [review]

**Comment by h1g0** [comment]
@d2dyno1 
> @h1g0 Can you add your agreement to #7625

I have already commented [here](https://github.com/files-community/Files/issues/7625#issuecomment-1013233428) in agreement. Please let me know if I need to respond further.

**Comment by d2dyno1** [comment]
Ah didn't know

**Review by d2dyno1 — APPROVED** [review]

**Pull request merged**